### ISSUE TITLE
Find patient screen should have a list of all patients (like a alphabetized list)

### ISSUE THREAD

**Issue by nehav39** [Intelehealth/intelehealth-openmrs-module#75]

**Comment by hardikk08** [comment]
https://talk.openmrs.org/t/gettting-all-results-from-rest-api-returns-empty-array/2622/2?u=hardikk08

So apparently the back end doesn't allows querying all the patients at once to avoid overloading the server. Reporting module is developed for that purpose as Darius said.

**Comment by nehav39** [comment]
Not possible.

**Comment by hardikk08** [comment]
I don't remember why we opened this. Querying the patient using alphabets also returns empty array. Patient can only be queried using uuid or minimum 3 letters from his/her name
The 2nd condition is what I figured out when trying to understand how find patient fragment works and by reading its code. Even rest return empty array on words less than 3 character.

![image](https://user-images.githubusercontent.com/29514954/39416918-37bf0104-4c6e-11e8-9368-ea1507d9660b.png)
